liquid_node/backup.py

```diff
@@ -258,18 +272,26 @@ def backup_collection_es(dest, name):
             else:
                 raise RuntimeError("Something went wrong: %r" % snapshot)
         log.info(f"Snapshot done in {int(time()-t0)}s")
-
-        backup_files(tmp_file, f"/es_repo/backup-{name}", [], SNOOP_ES_ALLOC)
+        if es_url == '/_h_es':
+            backup_files(tmp_file, f"/es_repo/backup-{name}", [], HYPOTHESIS_ES_ALLOC)
```

Comment:
This function will depend on whatever URL scheme is chosen outside it, too complex. This should be a separate function argument: `es_alloc_id`, and the caller chooses between the two constants.